kong/plugins/oauth2/secret.lua

```diff
@@ -0,0 +1,159 @@
+local utils = require "kong.tools.utils"
+
+
+local type = type
+local find = string.find
+local pcall = pcall
+local remove = table.remove
+local concat = table.concat
+local assert = assert
+
+
+--local CORES
+--do
+--  local infos = utils.get_system_infos()
+--  if type(infos) == "table" then
+--    CORES = infos.cores
+--  end
+--  if not CORES then
+--    CORES = ngx.worker.count() or 1
+--  end
+--end
```

Comment:
Is this a vestige from development?

Reply:
No, I left it there if we want Argon2 parallelism be adjusted by the number of cores. It is there if we at somepoint decide to do so.